R/graph.curve.params.R

```diff
@@ -7,7 +7,8 @@
 #' @param antigen_isos antigen isotypes
 #' @param alpha_samples `alpha` parameter passed to [ggplot2::geom_line]
 #' (has no effect if `show_all_curves = FALSE`)
-#' @param show_quantiles whether to show point-wise (over time) quantiles
+#' @param quantiles Optional numeric vector of quantiles to plot
+#' (e.g., `c(0.1, 0.5, 0.9)`). If `NULL`, no quantile lines are shown.
```

Comment:
Here you could specify what each of the numbers in the vector means for help. For example, 
```suggestion
#' (e.g., 10%, 50%, and 90% = `c(0.1, 0.5, 0.9)`). If `NULL`, no quantile lines are shown.
```